osquery/logger/CMakeLists.txt

```diff
@@ -61,3 +61,12 @@ if(NOT SKIP_AWS AND NOT WINDOWS)
   ADD_OSQUERY_LINK_ADDITIONAL("aws-cpp-sdk-sts")
   ADD_OSQUERY_LINK_ADDITIONAL("aws-cpp-sdk-core")
 endif()
+
+if(NOT WINDOWS)
```

Comment:
No you're.  I left it off on purpose because I wasn't sure if you guys wanted this to be built by default.  So I wasn't sure where it should be.  I'll make sure to utilize the `SKIP_AWS` :+1: